development/tools/jobs/kyma/test_integration_event_service_test.go

```diff
@@ -11,28 +11,28 @@ import (
 func TestGatewayTestsReleases(t *testing.T) {
 	for _, currentRelease := range tester.GetAllKymaReleaseBranches() {
```

Comment:
We need to ignore old releases i.e. 1.0, 1.1 and 1.2 by doing something like this:
```
unsupportedReleases := []tester.SupportedRelease{tester.Release10, tester.Release11, tester.Release12}

for _, currentRelease := range tester.GetKymaReleaseBranchesBesides(unsupportedReleases) {
```

Reply:
Done.